src/service/AnnounceService.ts

```diff
@@ -141,31 +156,49 @@ export class AnnounceService {
             }
             const nodePreset = (presetData.nodes || [])[index];
             const mainAccount = PublicAccount.createFromPublicKey(nodeAccount.main.publicKey, presetData.networkType);
+            if (operatingPublicKey) {
+                logger.info(
+                    `The Operating Account[public key: ${operatingPublicKey}] is creating transactions on behalf of your node account[public key: ${mainAccount.publicKey}].  Signers and cosigners may see a warning when signing the transactions on the Wallets!`,
```

Comment:
Could we unify the format in a method that knows how to print a (public)account? So we then define if we want to show public key, address or both in a centralized place. 

Even though address can be derived from a public key, addresses are easier to see and most people wouldn't be doing that public key -> address conversion